javagen/src/main/java/com/azure/autorest/template/ServiceVersionTemplate.java

```diff
@@ -0,0 +1,73 @@
+package com.azure.autorest.template;
+
+import com.azure.autorest.model.clientmodel.ServiceClient;
+import com.azure.autorest.model.javamodel.JavaFile;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ServiceVersionTemplate implements IJavaTemplate<ServiceClient, JavaFile> {
+    private static final ServiceVersionTemplate _instance = new ServiceVersionTemplate();
+
+    private String className;
+
+    private List<String> serviceVersions;
+
+    public static ServiceVersionTemplate getInstance() {
+        return _instance;
+    }
+
+    @Override
+    public void write(ServiceClient model, JavaFile javaFile) {
+        // imports
+        Set<String> imports = new HashSet<>();
+        imports.add("com.azure.core.util.ServiceVersion");
+        javaFile.declareImport(imports);
+
+        javaFile.javadocComment(comment -> {
+            comment.description("Service version");
```

Comment:
Could be improved to mention about the service?

Reply:
Added